### PR TITLE
test: small update to Storage test to get 100% code coverage

### PR DESCRIPTION
**Pull request by yusufaine**
image
The package's coverage was previously 100, 88, 95

Test didn't cover a small aspect on getUserPrefsPath which I assumed just had to be non-null.

### CODE REVIEW

**Review by blaketingyu — APPROVED** [review]
LGTM X 100 🌮 . Thanks for covering my storage.

File: src/main/java/seedu/address/commons/util/FileUtil.java [hunk 1]
```diff
@@ -45,12 +45,7 @@ public static void createIfMissing(Path file) throws IOException {
      * Creates a file if it does not exist along with its missing parent directories.
      */
     public static void createFile(Path file) throws IOException {
-        if (Files.exists(file)) {
-            return;
-        }
```
Comment (aidanaden) [inline src/main/java/seedu/address/commons/util/FileUtil.java:50]: Why was this line removed?

**Review by aidanaden — APPROVED** [review]
LGTM! tests seem p good only took 10000 months to approve this xd

wadiooooooooo 🍝
